src/lib/components/Select.react.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "./Select.css";
+/**
+ * Select is a dash wrapper for the html select tag.
+ */
+export default class Select extends Component {
+    handleChange(e) {
+        const options = e.target.selectedOptions;
+        const values = [];
+        for (let i = 0; i < options.length; i++) {
+            values.push(options[i].value);
+        }
+        this.props.setProps({ value: values });
```

Comment:
```suggestion
        this.props.setProps({ value: options.map(option => option.value) });
```
(not tested though...)

Reply:
I tried it but couldnt get map to work as options is not a javascript array, but a list of `HtmlOptions`